src/components/Calculator.js

```diff
@@ -1,70 +1,56 @@
-               <CalculatorFunction name={'+/-'} className="col-md-3 btn_number" add={this.addMe} id={'plusminus'} />
-               <CalculatorFunction name={'%'} className="col-md-3 btn_number" add={this.addMe} id={'percent'}/>
-               <CalculatorFunction name={'÷'} className="col-md-3 function" add={this.addMe} id={'divide'}/>
-               <CalculatorButton name="7" add={this.addMe} />
-               <CalculatorButton name="8" add={this.addMe}/>
-               <CalculatorButton name="9" add={this.addMe}/>
-               <CalculatorFunction name={'x'} add={this.addMe} id={'multiply'}/>
-               <CalculatorButton name="4" add={this.addMe}/>
-               <CalculatorButton name="5" add={this.addMe}/>
-               <CalculatorButton name="6" add={this.addMe}/>
-               <CalculatorFunction name={'-'} add={this.addMe} id={'minus'}/>
-               <CalculatorButton name="1" add={this.addMe}/>
-               <CalculatorButton name="2" add={this.addMe}/>
-               <CalculatorButton name="3" add={this.addMe}/>
-               <CalculatorFunction name={'+'} add={this.addMe} id={'plus'}/>
-               <CalculatorButton name={'0'} className="col-md-6 btn_number" add={this.addMe}/>
-               <CalculatorButton name="." className="col-md-3 btn_number " add={this.addMe} />
-               <CalculatorFunction name={'='} className="col-md-3 function" add={this.addMe} id={'equal'}/>
-            </div>
-    );
-  }
-}
\ No newline at end of file
+  return (
+    <div className='row calculator'>
+      <ResultView result={resultFormater()} />
+      <CalculatorFunction name={'AC'} className="col-md-3 btn_number" add={addMe} id={'ac'}/>
+      <CalculatorFunction name={'+/-'} className="col-md-3 btn_number" add={addMe} id={'plusminus'} />
+      <CalculatorFunction name={'%'} className="col-md-3 btn_number" add={addMe} id={'percent'}/>
+      <CalculatorFunction name={'÷'} className="col-md-3 function" add={addMe} id={'divide'}/>
+      <CalculatorButton name="7" add={addMe} />
+      <CalculatorButton name="8" add={addMe}/>
+      <CalculatorButton name="9" add={addMe}/>
+      <CalculatorFunction name={'x'} add={addMe} id={'multiply'}/>
+      <CalculatorButton name="4" add={addMe}/>
+      <CalculatorButton name="5" add={addMe}/>
+      <CalculatorButton name="6" add={addMe}/>
+      <CalculatorFunction name={'-'} add={addMe} id={'minus'}/>
+      <CalculatorButton name="1" add={addMe}/>
+      <CalculatorButton name="2" add={addMe}/>
+      <CalculatorButton name="3" add={addMe}/>
+      <CalculatorFunction name={'+'} add={addMe} id={'plus'}/>
+      <CalculatorButton name={'0'} className="col-md-6 btn_number" add={addMe}/>
+      <CalculatorButton name="." className="col-md-3 btn_number " add={addMe} />
+      <CalculatorFunction name={'='} className="col-md-3 function" add={addMe} id={'equal'}/>
+    </div>
+  );
+};
```

Comment:
- [ ] You did a great job on refactoring, but when I open the project in the browser It's not working as expected. When I use the `+, -, *, %, /` then it returns 0. like when I press `1 +` then it displays 0. Please fix it